From c2cdaa5c1515326e60fd3d7d48d920403d7a1695 Mon Sep 17 00:00:00 2001
From: Kexy Biscuit <kexybiscuit@aosc.io>
Date: Tue, 17 Dec 2024 05:45:49 +0800
Subject: [PATCH 309/314] AOSCOS: MIPS: add copyright headers back

In "FROMLIST: MIPS: Add syscall auditing support", a copyright header
was removed from arch/mips/include/uapi/asm/unistd.h with no proper
reason or related changes. Adding them back now.

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/include/uapi/asm/unistd.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/mips/include/uapi/asm/unistd.h b/arch/mips/include/uapi/asm/unistd.h
index b501ea16ccd4..63bfa241b3aa 100644
--- a/arch/mips/include/uapi/asm/unistd.h
+++ b/arch/mips/include/uapi/asm/unistd.h
@@ -6,6 +6,9 @@
  *
  * Copyright (C) 1995, 96, 97, 98, 99, 2000 by Ralf Baechle
  * Copyright (C) 1999, 2000 Silicon Graphics, Inc.
+ *
+ * Changed system calls macros _syscall5 - _syscall7 to push args 5 to 7 onto
+ * the stack. Robin Farine for ACN S.A, Copyright (C) 1996 by ACN S.A
  */
 #ifndef _UAPI_ASM_UNISTD_H
 #define _UAPI_ASM_UNISTD_H
-- 
2.48.1

